.eslintrc.js

```diff
@@ -42,7 +42,25 @@ module.exports = {
 				plugins: [ 'jsx' ],
 			},
 		},
+
 	},
+	overrides: [
+		{
+			files: [ '*.ts', '*.tsx' ],
+			extends: [
+				'plugin:@typescript-eslint/recommended',
+				// 'plugin:@typescript-eslint/recommended-requiring-type-checking',
+			],
```

Comment:
```suggestion
			extends: [
				'plugin:@typescript-eslint/strict',
			],
```

let's just go with strict?

Reply:
it will be a huge PR. Lets start with just recommended and then in other PR I'll make it more strict